tests/unit/ContinuationOut.spec.ts

```diff
@@ -40,6 +41,10 @@ describe('Continuation Out view', () => {
     businessStore.setLegalName('My Test Entity')
     businessStore.setIdentifier('CP1234567')
     businessStore.setFoundingDate('1971-05-12T00:00:00-00:00')
+    configurationStore.setConfiguration({
+      'VUE_APP_LEGAL_API_URL': 'https://legal-api.url/',
+      'VUE_APP_LEGAL_API_VERSION_2': 'v2'
+    })
```

Comment:
I prefer this style over the previous style (const configuration + setConfiguration(configuration)). It reads well this way. 👍